src/usr/local/www/interfaces_lagg_edit.php

```diff
@@ -263,6 +271,16 @@ function build_member_list() {
 ))->setHelp('Master interface for the <b>FAILOVER</b> mode. If auto is selected, then the first interface added is the master port; any interfaces added after that are used as failover devices.');
 $section->add($group);
 
+$group = new Form_Group('LACP Timeout Mode');
+$group->addClass('lacptimeout');
+$group->add(new Form_Select(
+	'lacptimeout',
+	'LACP Timeout',
+	$pconfig['lacptimeout'],
+	array('slow' => 'Slow (default)', 'fast' => 'Fast')
+))->setHelp('In a <b>Slow</b> timeout, PDUs are sent every 30 seconds and in a <b>Fast</b> timeout, PDUs are sent every second. LACP timeout occurs when 3 consecutive PDUs are missed. If LACP timeout is a slow timeout, the time taken when 3 consecutive PDUs are missed is 90 seconds (3x30 seconds). If LACP timeout is a fast timeout, the time taken is 3 seconds( 3x1 second).');
```

Comment:
Please break this up into multiple lines

Reply:
Fixed